src/OrbitGl/AccessibleThreadBar.cpp

```diff
@@ -46,9 +46,10 @@ orbit_accessibility::AccessibilityRect orbit_gl::AccessibleThreadBar::Accessible
   CHECK(canvas != nullptr);
 
   return orbit_accessibility::AccessibilityRect(
-      canvas->WorldToScreenWidth(local_pos[0]), canvas->WorldToScreenHeight(local_pos[1]),
-      canvas->WorldToScreenWidth(thread_bar_->GetSize()[0]),
-      canvas->WorldToScreenHeight(thread_bar_->GetSize()[1]));
+      canvas->GetViewport().WorldToScreenWidth(local_pos[0]),
+      canvas->GetViewport().WorldToScreenHeight(local_pos[1]),
+      canvas->GetViewport().WorldToScreenWidth(thread_bar_->GetSize()[0]),
+      canvas->GetViewport().WorldToScreenHeight(thread_bar_->GetSize()[1]));
```

Comment:
I've done that where appropriate now. At some point, this can probably disappear anyways because we should try to pass the Viewport instead of GlCanvas where possible.